docs/public/portal/assets/style.css

```
@@ -0,0 +1,167 @@
+  --panel-strong: rgba(20, 29, 54, 0.96);
+  --text: #e6eef8;
+  --muted: #9fb1c9;
+  --cyan: #33d6ff;
+  --blue: #5a8cff;
+  --violet: #a66cff;
+  --amber: #ffbf57;
+  --crimson: #ff4d6d;
+  --line: rgba(99, 214, 255, 0.24);
+  --line-strong: rgba(99, 214, 255, 0.46);
+  --shadow: 0 0 28px rgba(51, 214, 255, 0.12), inset 0 0 24px rgba(90, 140, 255, 0.05);
+  font-family: Inter, ui-sans-serif, system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", sans-serif;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)